examples/kafka-broker/verify.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -e
+
+export NAME=kafka-broker
+
+# shellcheck source=examples/verify-common.sh
+. "$(dirname "${BASH_SOURCE[0]}")/../verify-common.sh"
+
+run_log "Waiting for Kafka on 9092."
+while true; do if docker-compose logs -f kafka | grep -q 'started (kafka.server.KafkaServer)'; then
+    break;
+else
+    echo waiting ...; sleep 1;
+fi; done
+run_log "Kafka launched."
+
+# Topic
+TOPIC="envoy.kafka.broker"
+run_log "Creating topic."
+docker-compose exec kafka kafka-topics --bootstrap-server localhost:19092 --create --topic $TOPIC quickstart-events
+sleep 5
```

Comment:
Still need a loop for this check.

Reply:
Thank you for your point. I tested it many times and we don't need it anymore.